protocols/kad/src/protocol.rs

```diff
@@ -314,7 +314,7 @@ pub enum KadResponseMsg {
         /// The key of the record.
         key: record::Key,
         /// Value of the record.
-        value: Vec<u8>,
+        value: Bytes,
```

Comment:
I just explored this a bit more and `KadResponseMsg` is actually never cloned so we can probably just remove the trait and this here.